### PR TITLE
Feat/deliverable2

### PR DESCRIPTION
**Pull request by tomas10100**

### CODE REVIEW

**Comment by tomas10100** [comment]
ProductList page and filter added

File: src/pages/ProductList/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const [filters, setFilter] = useState([]);
+  const [productList, setProductList] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [showFilter, setShowFilter] = useState(false);
+
+  let loadingInterval;
+
+  useEffect(() => {
+    load();
+    return () => clearInterval(loadingInterval);
```
Comment (franciscogranadoswizeline) [inline src/pages/ProductList/index.js:25]: nice, i like that you apply the clear but...is interval o timeout what the code should implement?

**Review by franciscogranadoswizeline — APPROVED** [review]
good job tomas, just a left a comment. Check it out.

File: src/pages/ProductList/index.js (hunk 1 above)
Comment (tomas10100) [reply src/pages/ProductList/index.js:25]: I don't think so, but it's an easiest way to implement the requested, maybe a better way is with promises

**Pull request merged**